src/lib/Conversations.ts

```diff
@@ -50,4 +51,21 @@ export default class Conversations {
       }
     );
   }
+
+  async streamAllMessages(
+    callback: (message: DecodedMessage) => Promise<void>
+  ) {
+    XMTPModule.subscribeToAllMessages(this.client.address);
+    XMTPModule.emitter.addListener(
+      "message",
+      async (message: DecodedMessage) => {
+        if (this.known[message.id]) {
+          return;
+        }
+
+        this.known[message.id] = true;
+        await callback(new DecodedMessage(message));
```

Comment:
I think this can just be:
```suggestion
        await callback(message as DecodedMessage);
```